content/blog/manage-infrastructure-with-pac/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Manage Any Infrastructure with Policy as Code"
+date: 2020-02-18
+meta_desc: "Manage AWS, Azure, GCP, and Kubernetes with Policy as Code"
+meta_image: crossguard.png
+authors:
+   - sophia-parafina
+tags:
+   - "Policy as Code"
+   - "AWS"
+   - "Azure"
+   - "GCP"
+   - "Kubernetes"
+---
+ 
+In an  [earlier article]({{< relref "/blog/getting-started-with-pac" >}}), we introduced examples of Policy as Code to prevent two of the most common causes of data breaches. Policies are the guardrails of infrastructure. They control access, set limits, and manage how infrastructure operates. In many systems, policies are created by clicking on a GUI, making it difficult to replicate or version. Pulumi implements policy by writing it in Typescript, which ensures that you can write policies using software development practices such as automated testing, deployment, and version control.
```

Comment:
```suggestion
In an [earlier article]({{< relref "/blog/getting-started-with-pac" >}}), we introduced examples of Policy as Code to prevent two of the most common causes of data breaches. Policies are the guardrails of infrastructure. They control access, set limits, and manage how infrastructure operates. In many systems, policies are created by clicking on a GUI, making it difficult to replicate or version. Pulumi implements policy by writing it in TypeScript, which ensures that you can write policies using software development practices such as automated testing, deployment, and version control.
```